Nodejs/Product/TestAdapterImpl/TestFrameworks/Jasmine/jasmine.js

```diff
@@ -80,43 +81,59 @@ function initializeJasmine(Jasmine, projectFolder) {
     return instance;
 }
 
-function getSuiteName(spec) {
-    var suiteName = spec.getSpecName(spec);
-    if (suiteName && suiteName.length > 0) {
-        return suiteName.substr(0, suiteName.length - spec.description.length - 1);
-    }
-    return suiteName;
+/**
+ * @param {jasmine.Suite} suite
+ * @param {object[]} testList
+ * @param {string} testFile
+ */
+function enumerateSpecs(suite, testList, testFile) {
+    suite.children.forEach((child) => {
+        if (child instanceof jasmine.Suite) {
+            enumerateSpecs(child, testList, testFile);
+        } else {
+            testList.push({
+                test: child.getSpecName(child),
+                suite: suite.description,
+                file: testFile,
+                line: 0,
```

Comment:
The line and column don't matter?

Reply:
they do, but there is no way to get them from the Jasmine API.